python/lsst/ts/aos/utils/m1m3/m1m3_aav.py

```diff
@@ -361,6 +370,61 @@ def vect_to_fam(row: pd.Series) -> pd.Series:
         )
         self.calculated_fam.set_index(self.fitter.aav.index, inplace=True)
 
+    async def load_accelerometers(self, start: Time, end: Time) -> None | pd.DataFrame:
+        """Load DC accelerometers data in given interval. Those are used for
+        acceleration  corrections.
+
+        Parameters
+        ----------
+        start : Time
+            Search interval start time.
+        end : Time
+            Search interval end time.
+
+        Returns
+        -------
+        ret : pd.DataFrame
+            Time indexed dataframe with measured DC accelerometers values
+            (angularAcceleration[XYZ]).
+        """
+        logging.debug(f"Retrieving accelerometer data for {start.isot} - {end.isot}..")
+        ret = await self.client.select_time_series(
+            "lsst.sal.MTM1M3.accelerometerData",
+            ["timestamp"]
+            + [f"rawAccelerometer{acc}" for acc in range(8)]
+            + [f"accelerometer{acc}" for acc in range(8)]
+            + [f"angularAcceleration{a}" for a in "XYZ"],
+            start,
+            end,
+        )
+        if ret.empty:
+            logging.debug("empty, ignored")
+            return None
+        ret.set_index(
+            pd.DatetimeIndex(
+                Time(Time(ret["timestamp"], format="unix_tai"), scale="utc").isot
+            ),
+            inplace=True,
+        )
+        logging.debug(f"..OK ({len(ret.index)} records)")
```

Comment:
This comment is a little bit unclear, maybe "done loading accelerometer data (len datapoints)"?

Reply:
Changed, also similar HP notice:
```diff
@@ -406,7 +409,7 @@ class AccelerationAndVelocity:
             ),
             inplace=True,
         )
-        logging.debug(f"..OK ({len(ret.index)} records)")
+        logging.debug(f".. done loading accelerometer data, retrieved {len(ret.index)} records.")
         return ret
 
     async def collect_accelerometers_data(self) -> None:
@@ -444,7 +447,7 @@ class AccelerationAndVelocity:
             Time indexed dataframe with measured hardpoint forces (f[xyz]) and
             moments (m[xyz]).
         """
-        logging.debug(f"Retrieving HP data for {start.isot} - {end.isot}..")
+        logging.debug(f"Retrieving HP data for {start.isot} - {end.isot} ..")
         ret = await self.client.select_time_series(
```